### PR TITLE
SapMachine #299: Update SapMachine 12 ProblemLists with tests we see …

### LINKED ISSUE #299: Update SapMachine 12 ProblemLists with tests we see failing with jdk12 RC

### PR DESCRIPTION
**Pull request by GoeLin**
…failing with jdk12 RC

As against before, I added the excludes at the beginning of the files.

fixes #299

### CODE REVIEW

File: test/hotspot/jtreg/ProblemList-SapMachine.txt [hunk 1]
```diff
@@ -24,111 +24,35 @@
...
-# The remainder of this file is supposed to be identical to the internal
-# OpenJDK ProblemList.
-
 #############################################################################
 # Tests known to be failing in OpenJDK when jdk 11 was delivered (9/2018)
 # If one of these is fixed later, downport the fix to 11u!!
 
 # SapMachine 2016-01-12
-# Exclude because it occured the test crashed some server by consuming too much memory.
+# Exclude because it occured the test crashed some server (dewdfms0023) by consuming too much memory.
```
Comment (reshnm) [inline test/hotspot/jtreg/ProblemList-SapMachine.txt]: Remove internal host name.

File: test/jdk/ProblemList-SapMachine.txt [hunk 2]
```diff
@@ -126,50 +70,199 @@ jdk/jfr/event/oldobject/TestClassLoaderLeak.java
...
+# SapMachine 2019-02-08
+# RuntimeException: getBundle("UnreadableRB") doesn't throw class java.io.IOException
+java/util/ResourceBundle/Control/MissingResourceCauseTestRun.java                    windows-all
+
+# SapMachine 2019-02-08
+# Sporadic: OutOfMemoryError: Compressed class space
+java/lang/management/MemoryMXBean/LowMemoryTest2.sh                                  linux-s390x
+
+# SapMachine 2019-02-08
+# Both only failing on dewdfms0050. Do we need to fix the machine?
```
Comment (reshnm) [inline test/jdk/ProblemList-SapMachine.txt]: Remove internal server name.

**Comment by SapMachine** [comment]
Hello @GoeLin, this pull request fulfills all formal requirements.

**Comment by SapMachine** [comment]
Hello @GoeLin, this pull request fulfills all formal requirements.

**Pull request merged**